contents/docs/user-guides/funnels.md

```diff
@@ -4,74 +4,175 @@ sidebar: Docs
+
+A slightly more advanced approach is to use feature flags to run a simple A/B test, you can create a feature flag and assign 50% of people to the test group and 50% to the control group.
+
+You can then break down your funnel by this feature flag, you can then compare the conversion rate for the people in the test (feature flag on) and control (feature flag off) to see if there is a significant difference.
+
+N.B. When it comes to A/B testing - it’s important to recognize that there can be a lot of noise in experiments and if you have a very small improvement in your conversion rate is more likely to be due to some bias or noise than your feature being worse or better. Over time we will build out more advanced A/B testing capabilities with built-in statistical analysis to help you understand the confidence level in the results, check out our roadmap to learn more.
+
+
+### Understand seasonality in your conversion rates
+
+It’s unlikely that your conversion rate will remain stable every day or every week, throughout the year there will be external factors that cause your conversion rates to fluctuate - this is natural but it’s important to get a feel for them so you don’t jump to conclusions too quickly when analyzing a big change or drop-off.
+
+To look at the seasonality of your conversion rates click through to the historical trends view over the time period you might expect seasonal variations.
+
+A common seasonality to watch out for is weekends, particularly if your product is aimed at businesses it’s much more likely that they will use the product during the week than on the weekends. Perhaps you have a yearly sale such as “Black Friday” which increases the number of people coming to your site, but also increases the expectation of discounts.
+
+In this view you can adjust the date ranges to observe key seasonal trends and ensure when you look at your funnel you can tell whether it’s just a seasonal trend affecting it or whether something has materially changed in your product making the experience better or worse.
+
+
+### Refining your funnel to filter out noise
+
+When you’ve dialed in your funnel and you’re comfortable getting results and identifying the biggest drop-offs, you can take further steps to improve the signal to noise ratio.
+
+The most common configurations you might consider tweaking are:
+
+* **Conversion window limit:** This sets how long we should wait to consider someone a failure if they don’t achieve the success event - reducing this conversion window will ensure we see people you went through the funnel in a short space of time, increasing it will capture people who went part way through the funnel went a away and came back at a later time to complete it. Setting up an appropriate conversion window limit will also ensure your conversion rate is not changing over time as you’re measuring it (because of users who complete it much after).
```

Comment:
Switching from you to we. 

```suggestion
* **Conversion window limit:** This sets how long to wait before you consider someone a failure if they don’t achieve the success event. Reducing the conversion window will ensure you see who went through the funnel in a short space of time; increasing it will capture people who went part way through the funnel, went away and came back at a later time to complete it. Setting up an appropriate conversion window limit will also ensure your conversion rate is not changing over time as you’re measuring it.
```